src/systems/ec/implementations/vanilla/packages/ec-component-message/ec-component-message-print.scss

```diff
@@ -0,0 +1,55 @@
+/*
+ * Message
+ * @define message
+ */
+
+// Import base
+@import '@ecl/ec-base/ec-base';
+
+@mixin ecl-message-print($boder-width: 1.5pt) {
```

Comment:
I'd rather use `mm` or `cm` here

- typography: `pt`
- spacing and other measures: `mm`, `cm`

wouldn't it be better this way?